config/environments/production.rb

```diff
@@ -130,13 +130,12 @@
   config.action_mailer.perform_deliveries = true
   config.action_mailer.default_url_options = { host: protocol + ENV["APP_DOMAIN"].to_s }
   ActionMailer::Base.smtp_settings = {
-    address: "smtp.sendgrid.net",
-    port: "587",
-    authentication: :plain,
-    user_name: "apikey",
-    password: ENV["SENDGRID_API_KEY"],
-    domain: ENV["APP_DOMAIN"],
-    enable_starttls_auto: true
+    address: ENV["SMTP_ADDRESS"].presence || "smtp.sendgrid.net",
+    port: ENV["SMTP_PORT"].presence || 587,
+    authentication: ENV["SMTP_AUTHENTICATION"].presence || :plain,
+    user_name: ENV["SMTP_USER_NAME"].presence || "apikey",
+    password: ENV["SMTP_PASSWORD"],
```

Comment:
Not sure though: should this fallback to `ENV["SENDGRID_API_KEY"]` ?

Reply:
I originally planned to update config var everywhere else first then merge this PR. Thinking about this again, maybe the adding a fallback is a smoother process at the cost of one additional PR afterward.

```suggestion
    password: ENV["SMTP_PASSWORD"].presence || ENV["SENDGRID_API_KEY"],
```